src/components/Table.stories.mdx

```diff
@@ -0,0 +1,37 @@
+import { Canvas, Meta, Story, Props } from '@storybook/addon-docs';
+
+import { Table, TableBody, TableCaption, TableRow, TableHead, TableCell } from "./Table";
+import Caption from "./Caption";
+
+<Meta
+  title="Pure React components/Table"
+  component={Table}
+  argTypes={{
+    children: {table: {disable: true}},
+  }}
+  parameters={{
+    controls: {sort: 'requiredFirst'}
+  }}
+/>
+
+# Table
+
+export const Template = () => (
+  <Table>
+    <TableCaption>Table caption</TableCaption>
+    <TableBody>
+      <TableRow>
+        <TableHead scope="row">Header cell</TableHead>
+        <TableCell>Data cell</TableCell>
+      </TableRow>
```

Comment:
I notice the styling in Utrecht/Den Haag themes is off because of the borders that are present/absent for table cell/table head - maybe it'd be a bit more realistic to have a row with just thead and a row with just tcells? That would probably convey the intended usage better.

DH theme: 
![image](https://user-images.githubusercontent.com/5518550/194014752-17b5c157-8066-4adc-a19b-cad1ba72b862.png)